content/en/docs/user-documentation/layout-builder/donate/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Donate
+description: A call to action with donation buttons linking to an embedded form.
+---
+
+{{< tabpane text=true >}}
+  {{< tab header="Video" lang="vid" >}}
+    {{< youtube  >}}
+  {{< /tab >}}
+  {{% tab header="Diagram" lang="pic" %}}
+![A screenshot of the Donate block.](lb-donate--block.png)
+  {{% /tab %}}
+{{< /tabpane >}}
+
+**Designs:** [Mobile & Desktop](<../../../../../../assets/img/designs/lb-ui-kit/Donate.jpg>)
+
+The **Y Layout Builder - Donate** (`lb_donate`) and **YMCA Website Services Donation Embed Form** (`y_donate`) modules work together allow content editors to add an embedded donation form to the site and create a separate call to action to direct users there.
+
+## Embedded form
+
+The module works with donation forms from Blackbaud Online Express and Convio Luminate. To get started: 
+
+1. Enable the **YMCA Website Services Donation Embed Form** (`y_donate`) module at **Administration** > **Extend**.
+2. Select the **Layout** Tab of a Layout Builder-enabled page.
+3. Select **Add block** on the page, then search or scroll to find **Donation Form Embed Block**.
+4. Select the form type and enter the form ID from your donation provider.
+
+### Troubleshooting
+
+If your embedded form does not work in your non-production environment you may need to add a domain to the allow-list either on the provider-side or in your site's Content Security Policy.
+
+If your provider is not listed you can add the form by selecting the **Code** Custom Block and then pasting in your code. Alternatively you can work with your development partner to [add a new donation provider](https://git.drupalcode.org/project/y_donate#information-for-developers).
+
+## Donate Block
+
+The Donate Block can be placed in a full-width container.
```

Comment:
Let's explain or link what|how to place in `full-width` container

Reply:
Fixed in https://github.com/YCloudYUSA/yusaopeny_docs/commit/2816b0248fd0c0a1c968a718dbe56ed40881f4a5